### ISSUE TITLE
Manage post build jobs and process them in parallel

### ISSUE THREAD

**Issue by bruderol** [scenarioo/scenarioo#566]
As a scenarioo admin, I want to know what backgrond jobs are running and want to be able to see error logs.

Acceptance criterias
* Search Index calculation should not block import of builds
* DiffViewer calculations should not block import of builds
* all these potential "post jobs" should be displayed in an easy management UI to see what backgrond jobs are running and to see error information in case it fails (similar to normal build imports)
* not yet finished comparisons are not yet visible for selection
* if search index is not yet calculated the search box is not yet visible or an appropriate error message will inform the user that index is not yet calculated

Maybe / optional:
* resubmitting? (probably only through full build import?)
* canceling?

**Comment by bruderol** [comment]
@adiherzog @sascha-kaufmann @mi-we @cgrossde @eggerschwiler @mhanggi I put this out of scope for reelase 4.0 and move to next release - since not prio 1 for release 4.0  --- but if somebody wants to have some fun with background execution jobs to make build imports faster and not fail on elasticsearch problem - this might be a cool feature to implement. Just ask me, if you have questions.

**Comment by bruderol** [comment]
I thought about it once again --- if I understand right we never had any problems regarding the calculation of the serach index. it seems elasticsearch calls are fast enough. Therefore I deprioritize this and remove it from any milestone.

@mi-we @adiherzog agree?? Or did you hear of any project having troubles with this, that the index calculation made their imports too slow? If you hear about something like this, please add it again to next release.

**Comment by mi-we** [comment]
@bruderol Did not hear about any problems. I think it is a nice idea to decouple the ES indexing from the build import in regards to failures, but I agree on deprioritizing it.
